pkg/hhfctl/switch.go

```diff
@@ -236,3 +237,31 @@ func GetSerialInfo(sw *wiringapi.Switch) string {
 
 	return ""
 }
+
+func GetPowerInfo(ctx context.Context, swName string) map[string]string {
+	kube, err := kubeutil.NewClient(ctx, "", wiringapi.SchemeBuilder)
```

Comment:
it'll be better to pass kube client or switch object like in the `GetSerialInfo` b/c client creation could depend on the context and you probably would already have it initialized when calling this function

Reply:
Passing object should work fine as well

Reply:
I guess the annotations were not showing as I was adding them when the VLAB was already up. I'll change to the object approach and test again